plugin/src/addCustomGlobals.ts

```diff
@@ -0,0 +1,12 @@
+import { globals } from './globals';
+import type { ReanimatedPluginPass } from './types';
+
+// This function allows adding custom globals such as host-functions.
+
+export const addCustomGlobals = function (this: ReanimatedPluginPass) {
```

Comment:
What's the advantage of using an arrow function here instead of a function declaration (which seems to be the primary way of creating functions in JavaScript)?
```suggestion
export function addCustomGlobals(this: ReanimatedPluginPass) {
```

Reply:
This is not an arrow function but we could use simple function declaration regardless.